frontend/src/components/Plans/PlanDetails/PlanComponent.tsx

```diff
@@ -626,10 +626,10 @@ const PlanComponents: FC<PlanComponentsProps> = ({
                     ]}
                   />
                 </div>
-                <div className=" w-[96%] h-[1.5px] mt-8 mb-4 absolute bottom-16 bg-card-divider" />
-                <div className=" absolute bottom-[-4px] self-end">
+                <div className=" w-[96%] h-[1.5px] bg-card-divider" />
+                <div className="self-end">
```

Comment:
![Screen Shot 2023-03-03 at 9 07 35 AM](https://user-images.githubusercontent.com/33556500/222784449-9cc6c0f4-922d-4892-80de-9493645317b2.png)

Absolute was there because we still need all alerts/ dividers to appear in the same spot if there are multiple components.

If you check here you can see that the alert position is dependent on how many lines are above instead of having it always be based from the bottom.

Reply:
@gitstart @olalekanteeblaze